sakura_core/util/shell.cpp

```diff
@@ -38,63 +38,90 @@
 #include "env/CShareData.h"
 #include "env/DLLSHAREDATA.h"
 #include "extmodule/CHtmlHelp.h"
+#include <wrl.h>
 
-int CALLBACK MYBrowseCallbackProc(
-	HWND hwnd,
-	UINT uMsg,
-	LPARAM lParam,
-	LPARAM lpData
-)
+
+/*!
+	@brief IFileDialog の初期フォルダを設定する
+
+	@param [in] pDialog			設定対象のダイアログ
+	@param [in] pszInitFolder	初期フォルダに設定したいパス
+*/
+static void SetInitialDir( Microsoft::WRL::ComPtr<IFileDialog> pDialog, const WCHAR* pszInitFolder )
```

Comment:
別に正しく書く必要はありませんが、使い方が間違っています。

```suggestion
static void SetInitialDir( IFileDialog* pDialog, const WCHAR* pszInitFolder )
```

あとは好みですが、第二引数は std::wstring_view がいいっす。

Reply:
対応いたしました
ご指摘頂きありがとうございます